evap/evaluation/tests/test_models.py

```diff
@@ -140,6 +147,31 @@ def test_in_evaluation_to_published(self):
         evaluation = Evaluation.objects.get(pk=evaluation.pk)
         self.assertEqual(evaluation.state, Evaluation.State.PUBLISHED)
 
+    @override_settings(INSTITUTION_EMAIL_DOMAINS=["institution.example.com"])
+    def test_in_evaluation_to_published_with_external_participants(self):
+        # participant is external, since no institution email domain is set
+        participant = baker.make(UserProfile, email="foo@example.com")
+        evaluation = baker.make(
+            Evaluation,
+            state=Evaluation.State.IN_EVALUATION,
+            participants=[participant],
+            vote_start_datetime=datetime.now() - timedelta(days=2),
+            vote_end_date=date.today() - timedelta(days=1),
```

Comment:
For the test to pass between 0:00 and 3:00 (am), we need to use days=2 here (since the actual end is at 3am on the morning of the day after the "end" day, with 3 being configurable in the settings)